x-pack/platform/plugins/shared/task_manager/server/task_store.ts

```diff
@@ -156,6 +178,126 @@ export class TaskStore {
       maxRetries: 0,
     });
     this.requestTimeouts = opts.requestTimeouts;
+    this.security = opts.security;
+    this.canEncryptSavedObjects = opts.canEncryptSavedObjects;
+  }
+
+  public registerEncryptedSavedObjectsClient(client: EncryptedSavedObjectsClient) {
```

Comment:
If possible, would be nice to have these functions in another file / lib to avoid the task store class and test file from growing too much.